src/components/LocationPopover.vue

```diff
@@ -0,0 +1,53 @@
+<template>
+  <ion-chip outline>
+    <ion-icon :icon="locationOutline"/>
+    <ion-select interface="popover" :placeholder="$t('facility location')" :value="item.locationSeqId ? item.locationSeqId : facilityLocations[0].locationSeqId" @ionChange="setFacilityLocation($event)">
```

Comment:
If there are no facilityLocations then `facilityLocations[0].locationSeqId` will result in undefined error.